src/lib/fhirTables.js

```diff
@@ -397,3 +401,39 @@ function obsCompare(a, b) {
   return(effectiveB - effectiveA);
 }
 
+// +--------------------+
+// |    Plan of Care    |
+// +--------------------+
+
+
+function carePlanHeader() {
+  return (
+    <tr>
+      <th>Status</th>
+      <th>Intent</th>
+      <th>Activities</th>
+      <th>Category</th>
+      <th>Period Start</th>
+      {/* Add headers for other relevant CarePlan properties */}
+    </tr>
+  );
+}
+
+function carePlanRow(r, rmap, dcr) {
+  const status = r.status;
+  const intent = r.intent;
+  const activities = r.activity ? r.activity.map(activity => activity.detail.code.text).join(", ") : "";
+  const category = r.category ? r.category.map(c => c.text).join(", ") : "";
+  const periodStart = r.period ? futil.renderDateTime(r.period.start) : "";
```

Comment:
here use futil.renderPeriod(r.period) --- it'll do the right thing for all the random permutations of that type.

Reply:
applied `futil.renderPeriod(r.period)` -- thanks for pointing this out!